### ISSUE TITLE
Drastically simplify workshops/CONTRIBUTING.md

### ISSUE THREAD

**Issue by zachlatta** [hackclub/hackclub#725]
Can you drastically simplify `workshops/CONTRIBUTING.md` to reflect your experience contributing to workshops so far? And can you also add a "checklist" to make sure your workshop is ready for merge. Example (you'd want to include more elaboration in each step though):

```
- [ ] Markcop passes
- [ ] All filenames follow the right conventions (and the files are in the right place)
- [ ] The workshop has been added to the list in `workshops/README.md`
```

**Comment by jkwok91** [comment]
It is already pretty simple! Also, are there any simple instructions for running markcop?

**Comment by zachlatta** [comment]
If you have Docker installed, you can run the following command:

    $ docker run -it --rm -v $(pwd):/app hackclub/markcop

We don't have great instructions for running Markcop. The easiest thing to
do is probably to create a pull request and let CircleCI run the tests.
Maybe include documentation on doing that?

On Wed, Feb 17, 2016 at 9:25 PM, JESSICA <notifications@github.com> wrote:

> It is already pretty simple! Also, are there any simple instructions for
> running markcop?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/issues/725#issuecomment-185548076>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Comment by jkwok91** [comment]
Ok, I am adding a final section for what to do when the workshop is done.
"When workshop is done, make pull request. Make sure you have done x, y, z.
Then when you have made your PR, wait for circleci tests to report back,
etc."

2016-02-17 22:33 GMT-08:00 Zach Latta <notifications@github.com>:

> If you have Docker installed, you can run the following command:
>
> $ docker run -it --rm -v $(pwd):/app hackclub/markcop
>
> We don't have great instructions for running Markcop. The easiest thing to
> do is probably to create a pull request and let CircleCI run the tests.
> Maybe include documentation on doing that?
>

**Comment by zachlatta** [comment]
Sounds good. Make sure that someone who isn't very familiar with
contributing to open source can understand the instructions.

On Wed, Feb 17, 2016 at 10:38 PM, JESSICA <notifications@github.com> wrote:

> Ok, I am adding a final section for what to do when the workshop is done.
> "When workshop is done, make pull request. Make sure you have done x, y, z.
> Then when you have made your PR, wait for circleci tests to report back,
> etc."
>
> 2016-02-17 22:33 GMT-08:00 Zach Latta <notifications@github.com>:
>
> > If you have Docker installed, you can run the following command:
> >
> > $ docker run -it --rm -v $(pwd):/app hackclub/markcop
> >
> > We don't have great instructions for running Markcop. The easiest thing
> to
> > do is probably to create a pull request and let CircleCI run the tests.
> > Maybe include documentation on doing that?
> >
> > On Wed, Feb 17, 2016 at 9:25 PM, JESSICA <notifications@github.com>
> wrote:
> >
> > > It is already pretty simple! Also, are there any simple instructions
> for
> > > running markcop?
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> https://github.com/hackclub/hackclub/issues/725#issuecomment-185548076
> > >.
> > >
> >
> >
> >
> > --
> > Zach Latta
> > https://zachlatta.com <http://zachlatta.com>
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/hackclub/hackclub/issues/725#issuecomment-185562439
> >.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/issues/725#issuecomment-185564943>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Comment by zachlatta** [comment]
Closed by #728.
